zfsbootmenu/lib/zfsbootmenu-core.sh

```diff
@@ -1950,7 +1952,17 @@ zreport() {
   colorize orange "\n>> ZFSBootMenu commandline\n"
   get_zbm_kcl | kcl_assemble
 
-  colorize orange "\n\n>> ZFS/SPL module information\n"
+  colorize orange "\n\n>> Enabled hooks\n"
+  for hook in /libexec/hooks/*.d/*; do
+    [ -x "${hook}" ] && echo "* $( colorize green "${hook}")"
+  done
+
+  colorize orange "\n>> Disabled hooks\n"
+  for hook in /libexec/hooks/*.d/*; do
+    [ -x "${hook}" ] || echo "* $( colorize red "${hook}")"
```

Comment:
This is going to fail when there are no hooks installed.
```suggestion
    [ -f "${hook}" ] || continue
    [ -x "${hook}" ] && continue
    echo "* $( colorize red "${hook}")"
```